index.html

```diff
@@ -113,13 +113,13 @@
     //
     //-------------------------------------------------------------------------
 
```

Comment:
how many colours?

Reply:
Note: The problem here is that too many Tetris pieces are the same colour. In your review, you might want to suggest that I choose colours which would be easier to see when playing the game.